Xplat/src/main/java/vazkii/botania/common/item/WorldshaperssSextantItem.java

```diff
@@ -219,4 +307,35 @@ public static void render(PoseStack ms, Player player, ItemStack stack) {
 		}
 	}
 
+	@Override
+	public Component getName(@NotNull ItemStack stack) {
+		Component mode = Component.literal(" (")
+				.append(Component.translatable(getModeString(stack)))
+				.append(")");
+		return super.getName(stack).plainCopy().append(mode);
+	}
+
+	public static String getModeString(ItemStack stack) {
+		return "botaniamisc.sextantMode." + (getSphereMode(stack) ? "sphere" : "circle");
+	}
+
+	private static String getRadiusString(double radius) {
+		NumberFormat format = getNumberFormat(1);
+
+		return format.format(radius);
+	}
+
+	// these two methods should probably move to a Locale helper class:
```

Comment:
fine with keeping this one here, though I'd just inline the param value since it seems like you only pass `1`.